avatars/avatars.js

```diff
@@ -1981,7 +1981,7 @@ class Avatar {
     // update wind in simulation
     const _updateWind = () =>{
       const headPosition = localVector.setFromMatrixPosition(this.modelBoneOutputs.Head.matrixWorld);
-      wind.update(timestamp, headPosition, this.springBoneManager)
+      this.springBoneManager && wind.update(timestamp, headPosition, this.springBoneManager)
```

Comment:
Can you add a comment here to clarify that some avatars will not have spring bones?

Reply:
Added comment

https://github.com/webaverse/app/pull/3489/files